src/app/_components/TextInput/TextInput.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import TextInput from "./TextInput";
+
+export default {
+  title: "Components/TextInput",
+  component: TextInput,
+} as Meta<typeof TextInput>;
+
+const Template: StoryObj<typeof TextInput> = {
+  render: (args) => {
+    const [value, setValue] = useState(args.value);
```

Comment:
I think this useState should be inside the component since it should deal with its logic internally